src/extra/layouts/flex/lv_flex.c

```diff
@@ -93,10 +93,10 @@ void lv_flex_init(void)
 {
     LV_LAYOUT_FLEX = lv_layout_register(flex_update, NULL);
 
-    LV_STYLE_FLEX_FLOW = lv_style_register_prop();
-    LV_STYLE_FLEX_MAIN_PLACE = lv_style_register_prop() | LV_STYLE_PROP_LAYOUT_REFR;
-    LV_STYLE_FLEX_CROSS_PLACE = lv_style_register_prop() | LV_STYLE_PROP_LAYOUT_REFR;
-    LV_STYLE_FLEX_TRACK_PLACE = lv_style_register_prop() | LV_STYLE_PROP_LAYOUT_REFR;
+    LV_STYLE_FLEX_FLOW = lv_style_register_prop(0);
```

Comment:
```suggestion
    LV_STYLE_FLEX_FLOW = lv_style_register_prop(LV_STYLE_PROP_NONE);
```

I'm suggesting because `LV_STYLE_PROP_NONE` is more descriptive than `0`.